extensions/netty/deployment/src/main/java/io/quarkus/netty/deployment/NettyProcessor.java

```diff
@@ -253,11 +254,23 @@ RuntimeInitializedClassBuildItem runtimeInitBcryptUtil() {
     //if debug logging is enabled netty logs lots of exceptions
     //see https://github.com/quarkusio/quarkus/issues/5213
     @BuildStep
-    LogCleanupFilterBuildItem cleanup() {
+    LogCleanupFilterBuildItem cleanupUnsafeLog() {
         return new LogCleanupFilterBuildItem(PlatformDependent.class.getName() + "0", Level.TRACE, "direct buffer constructor",
                 "jdk.internal.misc.Unsafe", "sun.misc.Unsafe");
     }
 
+    /**
+     * On mac, if you do not have the `MacOSDnsServerAddressStreamProvider` class, Netty prints a warning saying it
+     * falls back to the default system DNS provider. This is not a problem and generates tons of questions.
+     *
+     * @return the log cleanup item removing the message
+     */
+    @BuildStep
+    LogCleanupFilterBuildItem cleanupMacDNSInLog() {
+        return new LogCleanupFilterBuildItem(DnsServerAddressStreamProviders.class.getName(), Level.WARN,
+                "Can not find io.netty.resolver.dns.macos.MacOSDnsServerAddressStreamProvider in the classpath");
```

Comment:
Should we make this thing a dependency on macOS instead? (Not sure what it brings)

Reply:
No, it's a native library, and cause more issue than anything.